### PR TITLE
docs: update for better readability

### PR DESCRIPTION
**Pull request by arthurlolico**
:)

### CODE REVIEW

File: onboarding/FIRST_PULL_REQUEST.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # My First PR
```
Comment (arantespp) [inline onboarding/FIRST_PULL_REQUEST.md:1]: Your First PR

File: onboarding/FIRST_PULL_REQUEST.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-### Hi, people! How are you doing? In this docs, I will write about the steps that I did to make my first PR. If any step is wrong or shallow, please give feedback. I'm counting on you. An observation: I will try to make the steps separate from the OS that I used.
+Hi, people! How are you doing? In this docs, we will learn about the steps to make our first PR. If any step is wrong or shallow, please give feedback. We're counting on you.
```
Comment (arantespp) [inline onboarding/FIRST_PULL_REQUEST.md:3]: In this doc

File: onboarding/FIRST_PULL_REQUEST.md [hunk 3]
```diff
@@ -11,35 +11,35 @@
...
 ### 1. Clone the repository using Git and create a new branch
 
-#### For the first part, it is necessary to clone the repository where you will work on your local PC. For this, open Git and type the following command:
+For the first part, it is necessary to clone the repository where we will work on your local PC. For this, open Git and type the following command:
 
-#### *git clone "repository of your choice"* 
+*git clone "repository of your choice"* 
 
-#### After this, you will create a new branch to work in the project. For this, you will use:
+After this, we will create a new branch to work in the project. For this, we will use:
```
Comment (arantespp) [inline onboarding/FIRST_PULL_REQUEST.md:18]: to work on the project

File: onboarding/FIRST_PULL_REQUEST.md [hunk 4]
```diff
@@ -11,35 +11,35 @@
...
 
-#### *git checkout -b "name of the issue"*
+*git checkout -b "name of the issue"*
 
 ### 2. Start working in the project
```
Comment (arantespp) [inline onboarding/FIRST_PULL_REQUEST.md:22]: Start working on the project

File: onboarding/FIRST_PULL_REQUEST.md [hunk 5]
```diff
@@ -11,35 +11,35 @@
...
 ### 3. Preparing for PR
 
-#### After the modifications are made, it's time for prepare to PR. Go to the root of your project, open Git and type this:
+After the modifications are made, it's time for prepare to PR. Go to the root of your project, open Git and type this:
 
-#### *git status*
+*git status*
 
-#### This command will highlight the files that were modified by you. If you want theses files on the PR, type:
+This command will highlight the files that were modified by you. If you want theses files on the PR, type:
```
Comment (arantespp) [inline onboarding/FIRST_PULL_REQUEST.md:32]: these files

File: onboarding/FIRST_PULL_REQUEST.md [hunk 6]
```diff
@@ -11,35 +11,35 @@
...
+*git add "each one of the files"*
 
-### After this, you have to save the project. For this, type:
+After this, we have to save the project. For this, type:
 
-#### *git commit -m "commentary about the update"*
+*git commit -m "commentary about the update"*
 
-#### This command saves the project and keep a commentary about the contents of the update. Finally, we have to upload the local repository. Type:
+This command saves the project and keep a commentary about the contents of the update. Finally, we have to upload the local repository. Type:
```
Comment (arantespp) [inline onboarding/FIRST_PULL_REQUEST.md:40]: the project and keeps a commentary

**Review by arantespp — CHANGES_REQUESTED** [review]
I prefer adding the reference near to the text in which it makes sense. For example, you can do

- [Click on Create Pull Request;](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)

Instead of using * to highlight a code, it's better to use three backticks. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

**Comment by arthurlolico** [comment]
Updated

**Comment by arantespp** [comment]
@arthurlolico, you didn't apply code blocks on commands 

> Instead of using * to highlight a code, it's better to use three backticks. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Also, you need to resolve all conversations.

**Review by arantespp — CHANGES_REQUESTED** [review]
See comment above

File: onboarding/FIRST_PULL_REQUEST.md (hunk 4 above)
Comment (arthurlolico) [reply onboarding/FIRST_PULL_REQUEST.md]: Fixed.

**Comment by arthurlolico** [comment]
Updated.

File: onboarding/FIRST_PULL_REQUEST.md (hunk 1 above)
Comment (arantespp) [inline onboarding/FIRST_PULL_REQUEST.md]: Instead of `My First PR`, change to `Your First PR`

File: onboarding/FIRST_PULL_REQUEST.md [hunk 7]
```diff
@@ -11,45 +11,41 @@
 
 ### 1. Clone the repository using Git and create a new branch
 
-#### For the first part, it is necessary to clone the repository where you will work on your local PC. For this, open Git and type the following command:
+For the first part, it is necessary to clone the repository where we will work on your local PC. For this, open Git and type the following command:
 
-#### *git clone "repository of your choice"* 
+*git clone "repository of your choice"*
```
Comment (arantespp) [inline onboarding/FIRST_PULL_REQUEST.md]: Add code block for all commands examples. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

**Review by arantespp — CHANGES_REQUESTED** [review]

**Comment by arthurlolico** [comment]
I put Your First PR and I used code block in all statements on the third commit, Pedro. Maybe I did something wrong?

**Review by arantespp — APPROVED** [review]

**Pull request merged**